example/pages/index.tsx

```diff
@@ -1,19 +1,23 @@
-import type { NextPage } from 'next';
-import Image from 'next/image';
-import UploadcareImage, { uploadcareLoader } from '@uploadcare/nextjs-loader';
-import { FC } from 'react';
-import styles from '../styles/Home.module.css';
+import type { NextPage } from 'next'
+import Image from 'next/image'
+import UploadcareImage, { uploadcareLoader } from '@uploadcare/nextjs-loader'
+import { FC } from 'react'
+import styles from '../styles/Home.module.css'
 
 type CodeProps = {
   [key: string]: any
 }
-const Code: FC<CodeProps> = (p) => 
-  <code className={styles.inlineCode} {...p} />
+const Code: FC<CodeProps> = (p) => <code className={styles.inlineCode} {...p} />
 
 const Home: NextPage = () => (
   <div className={styles.container}>
     <div className={styles.card}>
-      <h1>Uploadcare custom loader for Image Component <a href="https://github.com/uploadcare/nextjs-loader">@uploadcare/nextjs-loader</a></h1>
+      <h1>
```

Comment:
Seems like it's too much space here.

Reply:
Yeah, you're right. That's my default prettier config. I would add a prettier config to the project, but as separate PR.